packages/scrapers-v2/src/classify/README.md

```diff
@@ -0,0 +1,36 @@
+# Classify
+
+This folder contains code for the catalog entry classification stage of the scraper pipeline. In this stage, we classify
+catalog entries and annotate them with their type.
+
+## Classification Strategy
+
+We classify catalog entries by looking at the set of tabs (if it exists) at the top of the page. Here is a breakdown of
+the identifiable traits of our currently recognized entry types, by type:
```

Comment:
idk if im tripping but I think there's an extra line break after "breakdown of?" And there are some other ones in the later paragraphs